build_deploy.sh

```diff
@@ -4,7 +4,8 @@
 # Export vars for helper scripts to use
 # --------------------------------------------
 # name of app-sre "application" folder this component lives in; needs to match for quay
-export COMPONENT="hac-core"
+export COMPONENT="hac"
```

Comment:
I am a bit worried about the change from `hac-core` to `hac` but let's see how this rutns out.